server/queue/queue-pulsar/src/main/scala/org/apache/james/queue/pulsar/PulsarMailQueue.scala

```diff
@@ -97,6 +99,9 @@ class PulsarMailQueue(
   import serializers._
 
   type MessageAsJson = String
+  val DATE: Instant = Instant.parse("2023-04-14T10:00:00.00Z")
+  private val CLOCK: UpdatableTickingClock = new UpdatableTickingClock(DATE)
+  private val now: LocalDateTime = LocalDateTime.now(CLOCK)
```

Comment:
I use UpdatableTickingClock to test PulsarMailQueue whether Distributed test works or not. I will revert to now when test is success

Reply:
Inject a Clock here (just like Benoit injected Clock into MemoryMailQueue).

Reply:
Wrong suggestion, we likely can not manipulate clock of Pulsar. May need a diff way to test delay with Pulsar...